src/api/public/apidocs-new/paths/source_project_name_cmd_lock.yaml

```
@@ -0,0 +1,32 @@
+post:
+  summary: Locks the project
```

```suggestion
  summary: Locks the project.
```
Missing dot at the end of the sentence.

---

done